libraries/chain/include/eosio/chain/sync_call_context.hpp

```diff
@@ -8,14 +8,18 @@ namespace eosio { namespace chain {
 // When a new flag is added, its enum value must be 1 bit left shift from the last flag.
 // Update all_allowed_bits to include the newly added enum value.
 enum class sync_call_flags : uint64_t {
-   read_only        = 1ull<<0,
-   all_allowed_bits = read_only
+   // `force_read_only` is a user's directive to the system, telling the system whether
+   // the new call context created must operate in read-only mode or if it is
+   // free to operate under its most permissible mode.
```

Comment:
imo this description is missing a key point that with the flag clear it inherits the "readonlyness" from the calling context

Reply:
Thanks. I will reword the comments.

Reply:
https://github.com/AntelopeIO/spring/pull/1410/commits/6c799e6cd55f01a59b9bc22e2e575cd243a15dac